ruma-events/src/enums.rs

```diff
@@ -117,6 +125,10 @@ pub enum AnyRoomEventStub {
     Message(AnyMessageEventStub),
     /// Any state event stub
     State(AnyStateEventStub),
+    /// Any message event stub that has been redacted.
+    RedactedMessage(AnyRedactedMessageEventStub),
+    /// Any state event stub that has been redacted.
+    RedactedState(AnyRedactedStateEventStub),
 }
 
```

Comment:
Maybe something for a followup PR too, but we do now need enums over `AnyStateEvent | AnyRedactedStateEvent`, `AnyStateEventStub | AnyRedactedStateEventStub`, `AnyStrippedEvent | AnyRedactedStrippedEvent` and `AnyStrippedStateEventStub | AnyRedactedStrippedStateEventStub`. Maybe we actually need to use a `Regular` prefix for the generated `Message` and `State` enums and reuse their previous named for these new enums.

Reply:
I think the rename makes sense. Probably be best to do it as a new PR, and maybe after the `format_ident` and fixing of the brittle enum name generation (`ident.to_string().replace...`?

Reply:
Sounds good.